update manual for 3.3.0 release (#2)
Sensirion/myAmbienceManual: docs/README.md

## Requirements and Permissions

The following permissions need to be set so that MyAmbience can detect surrounding demonstrators:

- Bluetooth on your device needs to be enabled and the app needs to have permissions for bluetooth.
- On Android devices location permissions need to be given in addition. We do not store location information, however this is required by the bluetooth functionality since on some Android versions these are coupled.

> - In case you would like to get notifications when you download data from the demonstrators devices and export data, please allow notifications for MyAmbinece.

Reviewer: ```suggestion
- In case you would like to get notifications when you download data from the demonstrators devices and export data, please allow notifications for MyAmbience.
```

### Gadget Settings

You can configure certain aspects of your gadget! By tapping the cog wheel icon next to the favorites icon on the bottom right you go into the settings view. An example is shown below for the SCD4x CO2 Gadget:

![Settings View](images/Settings.PNG ":size=40%") 

Through the settings view you can:
- Update the firmware of your gadget. The app will automatically detect a firmware which needs to be updated and will allow you to start the update.
- Change the name of your gadget. Please note, that this will change the gadget name only for your own device and not overall of the gadget
- See the battery percentage, the available logging memory and adjust the logging interval 
- (CO2 Gadget Only) Adjust the LED Brightness
  - Turn on/off the LED by tapping on the light bulb on the bottom right
- (CO2 Gadget Only) Perform a Forced Recalibration

## Plots

> The plot view is very useful to understand the historical data from your environment. For example how is the CO2 concentration when you sleep at night with closed doors and windows or what is the relative humidity in the summer versus the winter.

Reviewer: ```suggestion
The plot view is very useful to understand the historical data from your environment. For example how the CO2 concentration is when you sleep at night with closed doors and windows or what the relative humidity is in the summer versus the winter.
```

There are two ways to initiate a data download:

- In the Plot screen, press the download icon on the top right which will take you to the download view and start fetching data for all selected gadgets.
- In the Dashboard, press the download icon on the gadget card. This will take you to the download view and start fetching data for this gadget.

The app needs to firstly connect to the gadget and then download the previously recorded data.

It is recommended that you are in close proximity of the gadget when trying to fetch the data.

![Download Data](images/DownloadsOverview.png ":size=40%")

On the download screen, you are informed about the download in progress. 
To cancel a running download, swipe the card to the left. Any data already downloaded is discarded.

As the download of data might take some time, you can leave the download view and the data download will continue in background. You are informed about

> the download progress with a notification. Additionaly, the download state is visualized with a icon in the bottom right of the gadgets's dashboard card.

Reviewer: ```suggestion
the download progress with a notification. Additionaly, the download state is visualized with an icon in the bottom right of the gadgets's dashboard card.
```

> You can go back to the download view tapping on that button at any time or using the download button in the Plot screen. 

Reviewer: ```suggestion
You can go back to the download view by tapping on that button at any time or by using the download button in the Plot screen. 
```

![Download Complete](images/DownloadsComplete.png ":size=40%")

> When a download finished, the download view shows you the number of samples downloaded and thier time range. You can explore the data in the Plot screen.

Reviewer: ```suggestion
When a download is finished, the download view shows you the number of samples downloaded and their time range. You can explore the data in the Plot screen.
```

> When a download failed or was interrupted due to loss or weak Bluetooth Connection, it is visualized in the download view. You can retry the download by pressing the refresh button on the right of a download card.

Reviewer: ```suggestion
When a download failed or was interrupted due to a lost or weak Bluetooth Connection, it is visualized in the download view. You can retry the download by pressing the refresh button on the right of a download card.
```

> If you do no longer want to keep the download card in the list, you can dismiss the card by swiping to the left or removing all downloads not in progress anymore with the "Dismiss fnished" button in the buttom right corner of the screen.

Reviewer: ```suggestion
If you do no longer want to keep the download card in the list, you can dismiss the card by swiping to the left or removing all downloads not in progress anymore with the "Dismiss finished" button in the bottom right corner of the screen.
```

## Export and Share data

You can export the stored data from your gadgets.

To select the data to export, you have two options:

> Use the share button on the top right in the Plot screen to export data for all selected.

Reviewer: ```suggestion
Use the share button on the top right in the Plot screen to export data for all selected gadgets.
```

![Share Data from Plot](images/Share.PNG ":size=10%")

Go to the [Manage Gadgets](#manage-gadgets) Menu, select one or more gadgets and choose the export option.

![Share Data from Manage Gadgets](images/ShareFromManage.png ":size=10%")

> Once you started a export, the Share exported data screen will open and show the progress of the export.

Reviewer: ```suggestion
Once you have started a export, the Share exported data screen will open and show the progress of the export.
```

> You can leave this screen and the export will continue in the background. You will be notified once the export finished.

Reviewer: ```suggestion
You can leave this screen and the export will continue in the background. You will be notified once the export is finished.
```

To navigate back to the export screen, use the [Share exported data](#share-exported-data) menu.

![Share exported data](images/Export%20Overview.PNG ":size=40%")

Once the export finished, you can share your data by tapping on the share button on the card.

The exported data persists until you dismiss it by swiping the entry to the left or you close the app.

> The data is exported in an EDF file format. The EDF format is CSV compatible which means you can open it with Excel or any other csv handling capable editor.

Reviewer: ```suggestion
The data is exported in an EDF file format. The EDF format is CSV compatible which means you can open it with Excel or any other editor capable of handling CSV.
```
